release/src/release-notes.ts

```diff
@@ -225,19 +227,19 @@ export const generateReleaseNotes = ({
   return releaseTemplate
     .replace(
       "{{enhancements}}",
-      groupIssuesByProductCategory(issuesByType.enhancements) ?? "",
+      formatIssues(issuesByType.enhancements),
```

Comment:
nit: the issues are already grouped, this function is really just doing the formatting.